deployment/ccip/changeset/cs_rmn_curse_uncurse.go

```diff
@@ -0,0 +1,275 @@
+package changeset
+
+import (
+	"encoding/binary"
+	"fmt"
+	"slices"
+
+	"github.com/pkg/errors"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	commoncs "github.com/smartcontractkit/chainlink/deployment/common/changeset"
+)
+
+func GlobalCurseSubject() Subject {
+	return Subject{0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01}
+}
+
+type RMNCurseAction struct {
+	ChainSelector  uint64
+	SubjectToCurse Subject
+}
+
+type CurseAction func(e deployment.Environment) []RMNCurseAction
+
+type RMNCurseConfig struct {
+	MCMS         *MCMSConfig
+	CurseActions []CurseAction
+	Reason       string
+}
+
+func (c RMNCurseConfig) Validate(e deployment.Environment) error {
+	state, err := LoadOnchainState(e)
```

Comment:
Since this is loaded in the changeset func call itself, maybe just pass in the state instead?

Reply:
I was trying to be compliant with: https://smartcontract-it.atlassian.net/browse/CCIP-4717